owncloudApp/src/main/java/com/owncloud/android/datamodel/ThumbnailsCacheManager.java

```diff
@@ -75,7 +75,7 @@
 
     public static Bitmap mDefaultImg =
             BitmapFactory.decodeResource(
-                    MainApp.getAppContext().getResources(),
+                    MainApp.Companion.getAppContext().getResources(),
```

Comment:
Use context in the constructor or the added independency.

Apply for all changes. It's not a good idea to have context as companion ;-)

Reply:
This change will be addressed in https://github.com/owncloud/android/issues/2351